mk/ml.rc.in

```diff
@@ -0,0 +1,43 @@
+#include <Windows.h>
+
+1 VERSIONINFO
+    FILEVERSION    @ML_VERSION@,@ML_PATCH@
+    PRODUCTVERSION @ML_VERSION@,@ML_PATCH@
+    FILEFLAGSMASK  VS_FFI_FILEFLAGSMASK
+    FILEFLAGS      (0 | @ML_FILEFLAGS@)
+    FILEOS         VOS_NT_WINDOWS32
+    FILETYPE       @ML_FILETYPE@
+BEGIN
+    BLOCK "StringFileInfo"
+    BEGIN
+        BLOCK "080904E4"
+        BEGIN
+#if ML_FILEFLAGS
+            VALUE "Comments", "Build @ML_BUILD_STR@  by  @ML_USER@"
+#else
+            VALUE "Comments", "Build @ML_BUILD_STR@"
+#endif
+            VALUE "CompanyName", "Elasticsearch BV"
+            VALUE "FileDescription", "@ML_FILENAME@ ( part of Elasticsearch machine learning )"
+            VALUE "FileVersion", "@ML_VERSION_STR@"
+            VALUE "InternalName", "@ML_NAME@"
+            VALUE "LegalCopyright", "Copyright (c) @ML_YEAR@ Elasticsearch BV"
+#if 0
+            VALUE "LegalTrademarks", "TODO"
+#endif
+            VALUE "OriginalFilename", "@ML_FILENAME@"
+#if ML_FILEFLAGS

```

Comment:
I think this logic will need changing now. Previously we were defining a load of preprocessor symbols when compiling this file but we aren’t now. So the check needs to be on something that CMake substitutes.